refs/debug.c

```diff
@@ -419,6 +419,12 @@ static int debug_reflog_expire(struct ref_store *ref_store, const char *refname,
 	return res;
 }
 
+static int debug_fsck(struct ref_store *ref_store,
+		      struct fsck_refs_options *o)
+{
+	return 0;
```

Comment:
It would be nice to add a trace entry here, look at the other functions.

Reply:
I will handle this.